funcx_endpoint/funcx_endpoint/executors/high_throughput/interchange.py

```diff
@@ -788,9 +839,13 @@ def start(self, poll_period=None):
                         r = pickle.loads(b_message)
                         logger.debug("[MAIN] Received result for task {} from {}".format(r, manager))
                         task_type = self.containers[r['container_id']]
-                        if r['task_id'] in self.task_status_deltas:
-                            del self.task_status_deltas[r['task_id']]
-                        self._ready_manager_queue[manager]['tasks'][task_type].remove(r['task_id'])
+                        logger.debug(f"[MAIN] Removing for manager:{manager} from {self._ready_manager_queue}")
+                        try:
+                            if r['task_id'] in self.task_status_deltas:
+                                del self.task_status_deltas[r['task_id']]
+                            self._ready_manager_queue[manager]['tasks'][task_type].remove(r['task_id'])
+                        except Exception:
+                            logger.exception("[MAIN] Caught exception here")
```

Comment:
what a terrible `except` block.

Reply:
This is obsolete, I've removed this handling.